devenv.nix

```diff
@@ -141,35 +143,41 @@ let
     TARGETFOLDER="${config.env.DEVENV_STATE}/importdb"
 
     rm -rf "$TARGETFOLDER"
-
     set -e
 
     if [[ "$1" == *.sql ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.gz ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.gz"
-        gunzip -c "$TARGETFOLDER/latest.sql.gz" > "$TARGETFOLDER/dump.sql"
-    elif [[ "$1" == *.sql.zip ]]; then
-        curl --create-dirs "$1" --output "$TARGETFOLDER/latest.sql.zip"
-        unzip -j -o "$TARGETFOLDER/latest.sql.zip" '*.sql' -d "$TARGETFOLDER"
+      ${pkgs.curl}/bin/curl --create-dirs "$1" --output "$TARGETFOLDER/dump.sql"
```

Comment:
If we always used `latest.sql` as target file, the logic for SQL_FILE could be simplified.

Reply:
The SQL File inside of the archives could be named differently. But ive renamed dump.sql to latest.sql for a small optimization.